src/components/dashboard/sidebar/DashboardSidebar.jsx

```diff
@@ -32,25 +32,32 @@ class DashboardSidebar extends React.Component {
   }
 
   renderOfferSummaryCard() {
-    const { offersCount } = this.props;
-    return (
-      <SidebarCard
-        title="Assigned courses left to redeem"
-        buttonText="Find a course in the catalog"
-        textClassNames={offersCount ? 'big-number' : ''}
-        buttonLink="/search"
-        linkIsLocal
-      >
-        {offersCount || 'You currently have no coupons to redeem.'}
-      </SidebarCard>
-    );
+    const { enterpriseConfig: { name } } = this.context;
+    const { offersCount, isOffersLoading } = this.props;
+    if (isOffersLoading) {
+      return (
+        <div className="mb-5">
+          <LoadingSpinner screenReaderText={`loading learning benefits for ${name}`} />
+        </div>
+      );
+    }
+    if (offersCount > 0) {
+      return (
+        <SidebarCard
+          title="Assigned courses left to redeem"
+          buttonText="Find a course in the catalog"
+          textClassNames={offersCount ? 'big-number' : ''}
+          buttonLink="/search"
+          linkIsLocal
+        >
+          {offersCount}
+        </SidebarCard>
+      );
+    }
+    return null;
```

Comment:
just to confirm: previously we told people "'You currently have no coupons to redeem.'" if they had 0 offers, but now it looks like we'd just return no card no nothing. I just want to confirm that is intended (and I'm reading this right)?

Reply:
That is correct, after talking with titans, we decided that the "no coupons" card was confusing for subscription users.